README.md

```diff
@@ -9,17 +8,24 @@ An example output from OptView is shown below.
 
 ![Example](doc/OptView.png)
 
-Documentation
--------------
+## Optimizer Support
+pyOptSparse provides Python interfaces for a number of optimizers.
+However, we do not provide the source code for SNOPT and NLPQLP, due to their restrictive license requirements.
+Please contact the authors of the respective codes if you wish to obtain them.
+Furthermore, ParOpt and IPOPT are available as a open source package but must be installed separately.
+Please see the documentation page of each optimizer for installation instructions.
 
+## Documentation
 Please see the [documentation](http://mdolab.engin.umich.edu/docs/packages/pyoptsparse/doc/index.html) for installation details and API documentation.
 
 To locally build the documentation, enter the `doc` folder and enter `make html` in terminal.
 You can then view the built documentation in the `_build` folder.
 
-Citation
---------
+## Testing
+Testing is done with the `testflo` package developed by the openMDAO team, which can be installed via `pip install testflo`.
+To run the tests, simply type `testflo .` in the root directory.
 
+## Citation
 Please cite pyOpt and the authors of the respective optimization
 algorithms in any publication for which you find it useful.
 For more background, theory, and figures, see the [pyOptSparse journal article](http://mdolab.engin.umich.edu/sites/default/files/pyOpt.pdf).
```

Comment:
I want this to be more clear that pyOpt and pyOptSparse are separate products, and this is probably the best place to do that